fiatconnect-api.md

```diff
@@ -2430,7 +2430,7 @@ If `keyType` is CPF, `key` MUST be a [CPF number](https://en.wikipedia.org/wiki/
 
 If `keyType` is `PHONE`, `key` MUST be an 11-digit [Brazilian mobile phone number](https://en.wikipedia.org/wiki/Telephone_numbers_in_Brazil). Area code MUST be included. The string MUST match the regex `/[0-9]{11}/` (a series of 11 digits, 0 through 9).
 
-Otherwise, if `keyType` is `RANDOM`, `key` MUST be a string of 32 characters consisting only of numbers 0-9, [ISO basic Latin alphabetic](https://en.wikipedia.org/wiki/ISO_basic_Latin_alphabet) letters, or [hyphen-minus](https://en.wikipedia.org/wiki/Hyphen-minus). If `keyType` is `RANDOM`, `key` represents a random key generated by the user's bank.
+Otherwise, if `keyType` is `RANDOM`, `key` MUST be a string of 36 characters consisting only of numbers 0-9, [ISO basic Latin alphabetic](https://en.wikipedia.org/wiki/ISO_basic_Latin_alphabet) letters, or [hyphen-minus](https://en.wikipedia.org/wiki/Hyphen-minus). If `keyType` is `RANDOM`, `key` represents a random key generated by the user's bank.
```

Comment:
```suggestion
Otherwise, if `keyType` is `RANDOM`, `key` MUST be [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier). If `keyType` is `RANDOM`, `key` represents a random key generated by the user's bank.
```

Reply:
it looks like it's actually supposed to be a UUID, which has even more constraints than the ones we wrote here-- for example, having five groups separated by 4 hyphens.

Reply:
Yes I think that's it